.github/workflows/release-fleet.yml

```diff
@@ -0,0 +1,86 @@
+name: release-fleet
+on:
+  workflow_dispatch:
```

Comment:
can we limit this to only be an option from release branches (pattern) or at least not allow it to work from main (primarily due to the lack of rule filtering of `> max_stack_version`

Reply:
good concern. the workflow works wherever it lives, and a workflow is a collection of jobs.
within the `fleet-pr` job, (L18), I think I can make a negate check that it will only work if the current branch is not `main`

Reply:
Can't easily limit where the job can run, but once it starts we can validate the branch.
Here's an example failure where `main` is forbidden: https://github.com/rw-access/detection-rules/runs/3071323321?check_suite_focus=true


Working as ofn 371535b503cb6d577b766aa0173156aecc4fb0fa